### DISCUSSION TITLE
Does plugnmeet work on intranet without internet?

### DISCUSSION THREAD

**Discussion by mehthab3** [mynaparrot/plugNmeet-server#379] [General]
Hi,

i just wanted to know whether plugnmeet will work on intranet without internet if i made a server setup internally.

**Comment by jibon57** [comment]
This isn't an issue, I moved it into discussion.

**Reply by jibon57** [reply]
I never tried it in intranet but if you can setup with domain + SSL certificate then it may work. You can try.

**Reply by mehthab3** [reply]
But while installing its asking for domain name. Can i use any internal domain or virtual hostname instead of domain name with ssl? 

I tried giving localhost and ip but installation is aborted
